app/helpers/cart_items_helper.rb

```diff
@@ -0,0 +1,2 @@
+module CartItemsHelper
```

Comment:
if you don't use helpers, you can remove the files

Reply:
Done bhai.